### PR TITLE
(DOCSP-26928): IA Refresh '[Port] Create an App User (/users/create)'

### PR DESCRIPTION
**Pull request by davidhou17**
## Pull Request Info
- Add info about Authentication Providers and Identity (Create a new account, Link to an Existing Account)
- Port the existing Create an App page + add tabs for Admin API / CLI
https://www.mongodb.com/docs/atlas/app-services/users/create/

### Jira

- https://jira.mongodb.org/browse/DOCSP-26928

### Staged Changes

- [Create an App User](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26928/users/create)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/users/create.txt [hunk 1]
```diff
@@ -226,39 +345,67 @@ There is no such specific method to re-run a custom confirmation
 function. Instead, App Services has a method to trigger a re-run of whatever
 the current Email/Password user confirmation workflow happens to be. You
 can manually re-run the currently selected user confirmation flow using
-the :doc:`App Services Admin API </admin/api/v3>` or via the App Services UI:
+the the App Services UI or the :doc:`App Services Admin API </admin/api/v3>`:
```
Comment (mongomoe) [inline source/users/create.txt:348]:
```suggestion
the App Services UI or the :doc:`App Services Admin API </admin/api/v3>`:
```
Required fix

File: source/users/create.txt [hunk 2]
```diff
@@ -226,39 +345,67 @@ There is no such specific method to re-run a custom confirmation
...
+      #. Select the :guilabel:`Run User Confirmation` button in the dialogue
+         box that appears.
 
-If the re-run fails or the user's tokens expire again, you can re-run
-the confirmation function as many times as necessary.
+      #. If the operation succeeds, the banner at the top of the App Services
+         admin console should display a confirmation message. The user's
+         :guilabel:`User Status` changes from :guilabel:`Pending Confirmation`
+         to :guilabel:`Pending User Login`. Once the user logs in, they will
+         move into the active users list automatically.
```
Comment (mongomoe) [inline source/users/create.txt:374]:
```suggestion
         move into the active user's list automatically.
```

Does the active user owner the list? If so, it should be 'user's list'

File: source/users/create.txt [hunk 3]
```diff
@@ -1,41 +1,108 @@
...
+user account the first time a user authenticates through the provider. The only exception is 
+:ref:`email/password user authentication <create-email-password-user>`, which requires 
+registering and confirming a user before the user can authenticate.
 
 .. include:: /includes/apple-account-deletion-requirements.rst
 
+Identity
+--------
+
+Upon first login with an authentication provider, App Services creates a
```
Comment (mongomoe) [inline source/users/create.txt:27]: Let's be more direct here. We may want to consider, 'who is first logging in with an authentication provider'. I think a better way to format this sentence would be: 

"When **you** first log in with an authentication provider, ..."

Not required, but good supplemental reading about 2nd person usage (if you're interested): https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/

File: source/users/create.txt [hunk 4]
```diff
@@ -226,39 +345,67 @@ There is no such specific method to re-run a custom confirmation
...
+           https://realm.mongodb.com/api/admin/v3.0/groups/<groupId>/apps/<appId>/user_registrations/by_email/<email>/run_confirm
+
+      .. seealso::
+       
+         :admin-api-endpoint:`Admin API Documentation <operation/adminRerunPendingUserConfirmation>`
 
 Summary
 -------
 
+- For most providers, App Services automatically creates a user object the first time
```
Comment (mongomoe) [inline source/users/create.txt:399]: Can we mention that this isn't for email password users in this summary sentence.

**Review by mongomoe — COMMENTED** [review]
Great job, very close to LGTM just needs a couple smaller required fixes before LGTM. 

Also, can we link user identities for all providers? If not we should mention the providers that cannot be linked.

File: source/users/create.txt (hunk 2 above)
Comment (davidhou17) [reply source/users/create.txt:374]: The user doesn't own the list, but the wording is a bit confusing. I'll change `active users list` to `list of active users`.

**Comment by davidhou17** [comment]
@mongomoe Thanks for the suggestions! Just made those fixes in the most recent commit.

Responding to your question: I believe we can link user identities for any provider since we're  just adding existing credentials to the user object (i.e. linking would not require registering and confirming a new email/password user).

File: source/users/create.txt (hunk 2 above)
Comment (mongomoe) [reply source/users/create.txt]: Oh language. Thanks for the fix

**Review by mongomoe — APPROVED** [review]
LGTM 👍

**Pull request merged**